testsuite/MDAnalysisTests/topology/test_pdb.py

```diff
@@ -233,6 +235,7 @@ def test_PDB_hex():
     assert u.atoms[3].id == 100002
     assert u.atoms[4].id == 100003
 
+@pytest.mark.filterwarnings("ignore::UserWarning")
```

Comment:
As per @lilyminium's comment, what expected warnings are being filtered here?

Reply:
The PDB_metals popped up the UserWarnings for the elements attributes. So, these warnings are filtered here. I think tis happened because a few of the test PDBs would not have the elements column. But, I didn't want those cases to affect the test case for metals(which happens here), hence I filtered the UserWarnings. Is there any other way to go around this problem?

Reply:
Because `@pytest.mark.filterwarnings("error")` is set below it (i.e. make all warnings into errors), I'm not sure if blanket disabling UserWarnings adheres to the intended test behaviour.

Instead here I would suggest using `@pytest.mark.filterwarnings("ignore: attributes will not be populated.")`. Thankfully there's overlap between the two warnings 🙃